apps/src/applab/designElements/image.jsx

```diff
@@ -187,14 +194,19 @@ export default {
     }
     const objectFitValue = element.getAttribute('data-object-fit');
     if (objectFitValue) {
+      //
+      // NOTE: not passing forceObjectFitNow because IE will crash when it
+      // is called here and also within makeDraggable() - which is called while
+      // in parseScreenFromLevelHtml()
```

Comment:
I don't fully understand the root cause of this timing problem, but if you do, I wonder if we can come up with a more durable fix.  This seems brittle because it requires everyone who touches this code in the future to understand whether it's safe to pass this third parameter.

If the problem is `applabObjectFitImages` being called for the same element more than once per tick, maybe we should throttle that call internally, making it idempotent within a 1ms window or so.

If the problem is a conflict with the jQuery draggable/resizable system, maybe we should compose a `decorateElement` operation that ensures these changes are always applied in a compatible way.

@Bjvanminnen I think you've also done quite a bit of work on designMode.js, do you have any other ideas for how we could approach this?

Reply:
It is not entirely clear to me why the browser is crashing. Obviously, it shouldn't crash. If you look at the attached native code stack trace (see below) from within IE, that was my first clue. Somehow, an image element with SVG attached is going away. And yet, this happens during load. So, after lots of experimentation, it became clear that this happens during the `makeDraggable()` and potentially as a result of the "watchers" that were set up when we previously invoked the polyfill at init time. The `makeDraggable()` was already a painful lesson for me when I first set up this polyfill. It definitely interferes with the IMG elements, which is why I had to call the polyfill again at the end of `makeDraggable()`.

As far as gaining a perfect understanding of the crash, I think it will be difficult. This fix is the result of 2 hours of trial and error based on my clues about what was going on. If you remove any of the changes above, the crashes start happening again. I couldn't find any mention online of this crash or the potential for the crash with this polyfill. So, I think we have a perfect storm of buggy IE, a polyfill that messes with IMG elements, and jQuery also messing with IMG elements that causes a unique problem.

![mshtml crash call stack](https://user-images.githubusercontent.com/5429146/33688811-bc606f28-da91-11e7-85ee-33902b380132.PNG)